docs/content/policy-language.md

```diff
@@ -3772,80 +3661,6 @@ For a reference on JSON Schema please see: <http://json-schema.org/understanding
 
 For a tool that generates JSON Schema from JSON samples, please see: <https://jsonschema.net/home>
 
-## Strict Mode
-
-The Rego compiler supports `strict mode`, where additional constraints and safety checks are enforced during compilation.
-Compiler rules that will be enforced by future versions of OPA, but will be a breaking change once introduced, are incubated in strict mode.
-This creates an opportunity for users to verify that their policies are compatible with the next version of OPA before upgrading.
-
-Compiler Strict mode is supported by the `check` command, and can be enabled through the `--strict`/`-S` flag.
-
-```
--S, --strict enable compiler strict mode
-```
-
-### Strict Mode Constraints and Checks
-
-Name | Description                                                                                                                                                                                                                                                    | Enforced by default in OPA version
---- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
-Duplicate imports | Duplicate [imports](../policy-language/#imports), where one import shadows another, are prohibited.                                                                                                                                                            | 1.0
-Unused local assignments | Unused arguments or [assignments](../policy-reference/#assignment-and-equality) local to a rule, function or comprehension are prohibited                                                                                                                                   |
-Unused imports | Unused [imports](../policy-language/#imports) are prohibited.                                                                                                                                                                                                  |
```

Comment:
`Unused local assignments` and `Unused imports` are still strict-mode checks that aren't part of v1. We should keep this section.

Or was it moved somewhere?

Reply:
Added these 2 checks back. Others are in `v0-upgrade.md`.